resource-management/test/resourceRegionMgrSimulator/data/regionNodeEvents.go

```diff
@@ -243,6 +243,7 @@ func makeDataUpdate(changesThreshold int) {
 			// each node has 10 changes within this cycle
 			nodeCopy := node.Copy()
 			nodeCopy.ResourceVersion = strconv.FormatUint(rvToGenerateRPs, 10)
+			CurrentRVs[rvLoc] = rvToGenerateRPs
```

Comment:
This line should be added after line 254 because CurrentRVs[rvLoc] stores new RV for each RP.

Reply:
Additional bug fixed.